### PR TITLE
WFCORE-678 console-enabled="false" does not work in domain mode

### PR DESCRIPTION
**Pull request by luck3y**
Fixes WFCORE-678, and allows the console to be disabled in domain mode.

### CODE REVIEW

**Comment by dmlloyd** [comment]
OK to test

**Comment by dmlloyd** [comment]
Not sure I'm an admin :)

File: host-controller/src/main/java/org/jboss/as/host/controller/operations/HttpManagementAddHandler.java [hunk 1]
```diff
@@ -127,10 +125,15 @@ static void populateHostControllerInfo(final LocalHostControllerInfoImpl hostCon
...
-    public static void installHttpManagementServices(final RunningMode runningMode, final ServiceTarget serviceTarget, final LocalHostControllerInfo hostControllerInfo,
-                                                     final HostControllerEnvironment environment,
-                                                     boolean onDemand, boolean httpUpgrade, final ServiceRegistry serviceRegistry, final OptionMap options) {
+    static void installHttpManagementServices(final HostControllerEnvironment environment, final LocalHostControllerInfo hostControllerInfo, final OperationContext context,
+                                              final ModelNode model) throws OperationFailedException {
 
+        boolean httpUpgrade = HttpManagementResourceDefinition.HTTP_UPGRADE_ENABLED.resolveModelAttribute(context, model).asBoolean();
+        boolean onDemand = context.isBooting();
+        OptionMap options = createConnectorOptions(context, model);
+        ServiceRegistry serviceRegistry = context.getServiceRegistry(false);
```
Comment (bstansberry) [inline host-controller/src/main/java/org/jboss/as/host/controller/operations/HttpManagementAddHandler.java]: This is a pre-existing issue, but the param should be 'true', not 'false'. The caller intends to modify state.

**Comment by bstansberry** [comment]
It looks like this branch needs a rebase or something, as the commit list includes an unrelated commit from Kabir.

**Comment by bstansberry** [comment]
Closing in favor of #701.

**Pull request closed without merging**